app/internal/world_clock.py

```diff
@@ -0,0 +1,510 @@
+COUNTRY_NAME_KEY_IN_CITIES = "country"
+SUBCOUNTRY_NAME_KEY_IN_CITIES = "subcountry"
+CONTINENT_NAME_KEY = "Continent_Name"
+CONTINENT = "continent"
+COUNTRY = "country"
+PLACE = "place"
+CURRENT_TIME_KEY = "datetime"
+TIMEZONE_PARTS_MAP = {CONTINENT: 0, PLACE: -1}
+TIME_SCOPE_INDEX = 0
+FEEDBACK_INDEX = 1
+ARB_COUNTRY_TIMEZONE = -1
+PATH_SEPARETOR = "/"
+
+
+class MeetingTime(NamedTuple):
+    time: str
+    start: str
+    end: str
+    desirability: str
+
+
+PARTS_OF_THE_DAY_FEEDBACK = [
+    MeetingTime(
+        time="Early Morning",
+        start="05:00:00",
+        end="07:59:59",
+        desirability="Better not",
+    ),
+    MeetingTime(
+        time="Morning",
+        start="08:00:00",
+        end="10:59:59",
+        desirability="OK"
+    ),
+    MeetingTime(
+        time="Late morning",
+        start="11:00:00",
+        end="11:59:59",
+        desirability="OK",
+    ),
+    MeetingTime(
+        time="Early afternoon",
+        start="12:00:00",
+        end="12:59:59",
+        desirability="OK",
+    ),
+    MeetingTime(
+        time="Afternoon",
+        start="13:00:00",
+        end="15:59:59",
+        desirability="OK"
+    ),
+    MeetingTime(
+        time="Late afternoon",
+        start="16:00:00",
+        end="16:59:59",
+        desirability="Can be considered",
+    ),
+    MeetingTime(
+        time="Early evening",
+        start="17:00:00",
+        end="18:59:59",
+        desirability="Can be considered",
+    ),
+    MeetingTime(
+        time="Evening",
+        start="19:00:00",
+        end="20:59:59",
+        desirability="Better not",
+    ),
+    MeetingTime(
+        time="Night",
+        start="21:00:00",
+        end="23:59:59",
+        desirability="Better not",
+    ),
+    MeetingTime(
+        time="Late night",
+        start="00:00:00",
+        end="04:59:59",
+        desirability="Not possible",
+    ),
+]
+
+
+def normalize_continent_name(continent_name: str) -> Optional[str]:
+    """Normalize a given continent name to the continent name
+       that appears in the timezone list.
+    Args:
+        continent_name (str): The given continent name.
+    Returns:
+        str: The normalized continent name.
+    """
+    for continent in CONTINENTS:
+        if continent in continent_name:
+            return continent
+    return continent_name
+
+
+def load_country_continent_data_set() -> List[Dict[str, str]]:
+    """Load the country-continent data set.
+
+    Returns:
+        list: The country-continent data set
+    """
+    with open(COUNTRY_TO_CONTINENT_DATA_SET_PATH) as file:
+        return json.load(file)
+
+
+def get_continent(country_name: str) -> Optional[str]:
+    """Get the continent of a given country.
+    Args:
+        country_name (str): The given country name.
+    Returns:
+        str: The suitable continent name.
+    """
+    details = load_country_continent_data_set()
+    for country_element in details:
```

Comment:
Same here: put the name of the country as key to make the search much faster

Reply:
done